### PR TITLE
Fix spelling mistakes found by codespell

### PR DESCRIPTION
**Pull request by knweiss**
`codespell` can be found here: https://github.com/lucasdemarchi/codespell

Signed-off-by: Karsten Weiss <knweiss@gmail.com>

### CODE REVIEW

File: rules/manager_test.go [hunk 1]
```diff
@@ -221,7 +221,7 @@ func TestStaleness(t *testing.T) {
 	testutil.Equals(t, want, samples)
 }
 
-// Convert a SeriesSet into a form useable with reflect.DeepEqual.
+// Convert a SeriesSet into a form usable with reflect.DeepEqual.
```
Comment (jamtur01) [inline rules/manager_test.go]: IMHO This isn't a spelling mistake - both are acceptable - even in US English. :)

Comment (knweiss) [reply rules/manager_test.go]: @jamtur01 Removed.

**Comment by knweiss** [comment]
Why is Travis unsuccessful?

**Comment by simonpasquier** [comment]
This is a flaky test apparently.

**Comment by gouthamve** [comment]
Restarted it.

**Comment by krasi-georgiev** [comment]
this test has been bugging us for ages :(

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**